File: lua/test/exists_test.lua
-- BranchAggregate SDK exists test

local sdk = require("branch-aggregate_sdk")

describe("BranchAggregateSDK", function()
  it("should create test SDK", function()
    local testsdk = sdk.test(nil, nil)
    assert.is_not_nil(testsdk)
  end)
end)

File: php/test/ExistsTest.php
<?php
declare(strict_types=1);

// BranchAggregate SDK exists test

require_once __DIR__ . '/../branchaggregate_sdk.php';

use PHPUnit\Framework\TestCase;

class ExistsTest extends TestCase
{
    public function test_create_test_sdk(): void
    {
        $testsdk = BranchAggregateSDK::test(null, null);
        $this->assertNotNull($testsdk);
    }
}
